XenAdmin/Actions/GUIActions/DownloadUpdatesXmlAction.cs

```diff
@@ -293,6 +294,12 @@ protected virtual XmlDocument FetchCheckForUpdatesXml(string location)
             var xdoc = new XmlDocument();
             var uri = new Uri(location);
             var proxy = XenAdminConfigManager.Provider.GetProxyFromSettings(Connection, false);
+            
+            var uniqueIdHash = GetUniqueIdHash();
+
+            string buildText = Helpers.CommonCriteriaCertificationRelease
+                        ? string.Format("{0}: {1}", Program.Version.Revision, Messages.COMMON_CRITERIA_TEXT)
+                        : Program.Version.Revision.ToString();

```

Comment:
This just takes the build number. I think you want the three-digit version number instead.